core-models/src/main/scala/com/mesosphere/usi/core/models/StateEvent.scala

```diff
@@ -27,6 +27,7 @@ object StateSnapshot {
   */
 sealed trait StateUpdated extends StateEvent
```

Comment:
I don't quite understand why we need two sealed traits `StateUpdated` and `StateEvent`. What's the difference? /cc @timcharper

Reply:
`StateUpdated` seems redundant to me at this point. @timcharper ?